assets/scripts/lobby9.js

```diff
@@ -534,12 +534,18 @@ function teamLeaderSetup(phase) {
       $("#missionsBox").css("left", (widthOfRoom/2) + "px");
 }
 
-//set up the hover over missions box and highlight participating members
-
-
-
-
-
+//when hovering over missions box, highlight members participating in mission
+$("#missionsBox").hover(function(){
+    for (var i = 0; i < gameData.proposedTeam.length; i++) {
+		for (var i = 0; i < gameData.proposedTeam.length; i++) {
```

Comment:
Is this duplicate for loop a mistake?

Reply:
Yep, that was a mistake. It has been removed now!

Reply:
Awesome, remember to re-push your updates to your github branch. The changes will be automatically applied here too :).